pkg/kubectl/cmd/create_namespace.go

```diff
@@ -51,6 +51,8 @@ func NewCmdCreateNamespace(f *cmdutil.Factory, cmdOut io.Writer) *cobra.Command
 	cmdutil.AddValidateFlags(cmd)
 	cmdutil.AddPrinterFlags(cmd)
 	cmdutil.AddGeneratorFlags(cmd, cmdutil.NamespaceV1GeneratorName)
+	cmdutil.AddInclude3rdPartyFlags(cmd)
```

Comment:
You don't need this here.  You only need this for commands that are trying to create a `RESTMapper` that might actually need third party resources.  There are a few other commands that don't need the flag either.